<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>847</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RAMSIS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="leftAreaWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="TreeView" name="forecastTreeView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>180</number>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="footerAreaWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>256</width>
           <height>40</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="planNextButton">
            <property name="text">
             <string>Plan Next</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="addScenarioButton">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeScenarioButton">
            <property name="text">
             <string>––</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="mainAreaWidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QWidget" name="statusAreaWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #F8E81C;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="fcStatusLabel">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>No Scenario Selected</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QTabWidget" name="resultTabWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="elideMode">
          <enum>Qt::ElideRight</enum>
         </property>
         <property name="usesScrollButtons">
          <bool>false</bool>
         </property>
         <property name="documentMode">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="generalTab">
          <attribute name="title">
           <string>Status</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="topMargin">
            <number>20</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="scenarioTitleLabel">
             <property name="font">
              <font>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Please select a scenario</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0">
            <widget class="QWidget" name="stageStatusWidget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="1" rowspan="2">
            <widget class="QWidget" name="tlWidget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QWidget" name="bottomAreaWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: gray;
color:white;
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="topMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="TimeLinePlotWidget" name="timeLineWidget"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="styleSheet">
          <string notr="true">color:#707070;</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <widget class="QComboBox" name="timelineSelectionBox"/>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="showForecastsCheckBox">
            <property name="text">
             <string>Forecasts</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionApplication_Settings"/>
    <addaction name="actionProject_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionManage_Projects"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionStart_Simulation"/>
    <addaction name="actionPause_Simulation"/>
    <addaction name="actionStop_Simulation"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionShow_3D"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSimulation"/>
   <addaction name="actionShow_3D"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <action name="actionImport_Seismic_Data">
   <property name="text">
    <string>&amp;Import  from CSV...</string>
   </property>
  </action>
  <action name="actionView_Data">
   <property name="text">
    <string>View as Table</string>
   </property>
  </action>
  <action name="actionStart_Simulation">
   <property name="text">
    <string>Start Simulation</string>
   </property>
  </action>
  <action name="actionPause_Simulation">
   <property name="text">
    <string>Pause Simulation</string>
   </property>
  </action>
  <action name="actionStop_Simulation">
   <property name="text">
    <string>Stop Simulation</string>
   </property>
  </action>
  <action name="actionShow_3D">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/toolbar-buttons/images/view-3d-toolbar-button.png</normaloff>:/toolbar-buttons/images/view-3d-toolbar-button.png</iconset>
   </property>
   <property name="text">
    <string>3D View</string>
   </property>
  </action>
  <action name="actionImport_Hydraulic_Data">
   <property name="text">
    <string>Import from CSV...</string>
   </property>
  </action>
  <action name="actionManage_Projects">
   <property name="text">
    <string>Manage Projects...</string>
   </property>
  </action>
  <action name="actionSimulation">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/toolbar-buttons/images/simulate-toolbar-button.png</normaloff>:/toolbar-buttons/images/simulate-toolbar-button.png</iconset>
   </property>
   <property name="text">
    <string>Simulation</string>
   </property>
  </action>
  <action name="actionApplication_Settings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Application Settings...</string>
   </property>
  </action>
  <action name="actionScenario">
   <property name="icon">
    <iconset>
     <normaloff>:/toolbar-buttons/images/scenario-toolbar-button.png</normaloff>:/toolbar-buttons/images/scenario-toolbar-button.png</iconset>
   </property>
   <property name="text">
    <string>Scenario</string>
   </property>
   <property name="toolTip">
    <string>Compute Scenario</string>
   </property>
  </action>
  <action name="actionImport_FDSNWS_Data">
   <property name="text">
    <string>Import FDSNWS Data</string>
   </property>
  </action>
  <action name="actionFetch_from_fdsnws">
   <property name="text">
    <string>&amp;Fetch from FDSNWS</string>
   </property>
  </action>
  <action name="actionFetch_from_hydws">
   <property name="text">
    <string>Fetch from HYDWS</string>
   </property>
  </action>
  <action name="actionSeismic_Data">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>SEISMIC DATA</string>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
  </action>
  <action name="actionHydraulic_Data">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>HYDRAULIC DATA</string>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
  </action>
  <action name="actionDelete_Results">
   <property name="text">
    <string>Delete all forecast results...</string>
   </property>
  </action>
  <action name="actionProject_Settings">
   <property name="text">
    <string>Project Settings...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TimeLinePlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>RAMSIS.ui.views.widgets</header>
  </customwidget>
  <customwidget>
   <class>TreeView</class>
   <extends>QTreeView</extends>
   <header>RAMSIS.ui.base.tree.view</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
